giskard/scanner/issues.py

```diff
@@ -90,6 +133,10 @@ def __init__(
         self._features = features
         self._tests = tests
         self.taxonomy = taxonomy or []
+        self.display_footer_info = display_footer_info
+        self.scan_examples = DataFrameScanExamples() if scan_examples is None else scan_examples
+        if self._examples is not None:
```

Comment:
Do we still need to assign something to self._examples ?
We could just use the arguments here